### ISSUE TITLE
Comments containing PowerShell comments (e.g. "#...") are rendered as headings

### ISSUE THREAD

**Issue by jeremy-jameson** [technology-toolbox/website#88]
The markdown help for Commento doesn't mention anything about headings, but it renders `<h1>` elements when a line starts with `#`.

For example:

https://www.technologytoolbox.com/blog/jjameson/2013/05/24/powershell-scripts-for-managing-backconnectionhostnames-kb-896861/#commento-9f580c040367dc308b1dad81772cf414dce7339541ad2ac6312339b24a745dec

Due to the background image specified in the CSS, this currently renders like this:

![image](https://user-images.githubusercontent.com/1385288/141467370-a0dac178-9cd8-49d7-8a1c-cac3e379edc2.png)

Note that in this particular example, there is no space after the `#` character:

```
#Set the DisableStrictNameChecking...
```

In other words, there are a couple of issues here:

1. The formatting due to the `<h1>` element is an eyesore
2. Commento should strip heading tags from comments (since it doesn't really make sense to render headings within comments)

**Comment by jeremy-jameson** [comment]
Here is what that comment looks like after the CSS change:

![image](https://user-images.githubusercontent.com/1385288/141484609-62eddce5-8335-484a-a985-8e5534b2f052.png)

Note the comment is missing the `#` characters entered by the user (since those are mistakenly interpreted by Commento as H1 markdown), but I can live with that -- at least for now.
